### PR TITLE
Ratings fix driver

### PR DESCRIPTION
**Pull request by jonathanadler**

### CODE REVIEW

File: src/components/Ratings/Ratings.driver.ts [hunk 1]
```diff
@@ -57,15 +55,10 @@ export const ratingsDriverFactory = (base: UniDriver): RatingsDriver => {
       return getStarInput(idx - 1).hover();
     },
     async getHoveredStars() {
-      const icons = getStarIcons();
+      const hoveredCount = await base.attr(RATINGS_DATA_KEYS.Hovered);
+      console.log('adler', 'Ratings.driver.ts:59', hoveredCount);
```
Comment (nitayneeman) [inline src/components/Ratings/Ratings.driver.ts:59]: Redundant?

**Review by nitayneeman — APPROVED** [review]
Looks good

File: src/components/Ratings/Ratings.driver.ts (hunk 1 above)
Comment (jonathanadler) [reply src/components/Ratings/Ratings.driver.ts:59]: 🤦

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_492/

**Pull request merged**